### PR TITLE
LPS-156477 Add a test to can get ManyToMany relationship details as nested fields_4

### PR DESCRIPTION
**Pull request by Hijky**
**Background**
Add a test to can get ManyToMany relationship details as nested fields_4

**Test Plan**
Given university created
And Given subject created
And Given manyToMany relationship universitySubjects created
And Given university entry created
And Given subject entry created
When in c/subjects I use PUT /{subjectId}/universitiesSubjects/{universityId}
Then the university has the subject related in /o/c/universities?nestedFields=universitiesSubjects

**JIRA Ticket**
https://issues.liferay.com/browse/LPS-156477

### CODE REVIEW

**Comment by Hijky** [comment]
ci:test:headless-frontend-functional:nocompile

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/AdvancedObjectAPI/SupportManyToManyRelationships.testcase [hunk 1]
```diff
@@ -208,4 +208,62 @@ definition {
...
+				en_US_plural_label = "subjects",
+				objectEntry1 = "${subjectId}",
+				objectEntry2 = "${universityId}",
+				relationshipName = "universitiesSubjects");
+		}
+
+		task ("Then the subject is related to the university in subjects GET endpoint with nestedFields=universitiesSubjects") {
+			var getObjectsWithNestedFieldReponse = ObjectDefinitionAPI.getObjectsWithNestedField(
+				nestedField = "universitiesSubjects",
+				objects = "universities");
```
Comment (magjed4289) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/AdvancedObjectAPI/SupportManyToManyRelationships.testcase:251]: we want to use the `subjects GET endpoint` in this step, not the university.

**Review by magjed4289 — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/AdvancedObjectAPI/SupportManyToManyRelationships.testcase (hunk 1 above)
Comment (magjed4289) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/AdvancedObjectAPI/SupportManyToManyRelationships.testcase:251]: Please review 😅

Comment (Hijky) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/AdvancedObjectAPI/SupportManyToManyRelationships.testcase:251]: I edit it for your review

**Review by magjed4289 — APPROVED** [review]

**Comment by Hijky** [comment]
ci:forward

**Pull request closed without merging**